clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_2_0.xsd

```diff
@@ -287,7 +287,7 @@
     </xs:complexType>
 
     <xs:complexType name="locking">
-        <xs:attribute name="isolation" type="tns:isolation" default="REPEATABLE_READ">
+        <xs:attribute name="isolation" type="tns:isolation" default="READ_COMMITTED">
```

Comment:
The current parser doesn't care what the version was. The actual value will be the value which is currently set as default, see:

https://github.com/wildfly/wildfly/blob/master/clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java#L844

https://github.com/wildfly/wildfly/blob/master/clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/LockingResourceDefinition.java#L61

On the clustering call on Feb 4 we decided to go with changing the XSD only, as we have done in the past, so that legacy configurations will upgrade to current defaults.

https://issues.jboss.org/browse/WFLY-6124?focusedCommentId=13158837&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13158837